vector/src/main/java/im/vector/app/features/command/CommandParser.kt

```diff
@@ -78,7 +78,29 @@ object CommandParser {
                         ParsedCommand.ErrorSyntax(Command.CHANGE_DISPLAY_NAME)
                     }
                 }
-                Command.TOPIC.command                  -> {
+                Command.CHANGE_DISPLAY_NAME_FOR_ROOM.command -> {
+                    val newDisplayName = textMessage.substring(Command.CHANGE_DISPLAY_NAME_FOR_ROOM.command.length).trim()
+
+                    if (newDisplayName.isNotEmpty()) {
+                        ParsedCommand.ChangeDisplayNameForRoom(newDisplayName)
+                    } else {
+                        ParsedCommand.ErrorSyntax(Command.CHANGE_DISPLAY_NAME_FOR_ROOM)
+                    }
+                }
+                Command.CHANGE_AVATAR_FOR_ROOM.command       -> {
+                    if (messageParts.size == 2) {
+                        val url = messageParts[1]
+
+                        if (url.isNotEmpty()) {
+                            ParsedCommand.ChangeAvatarForRoom(url)
```

Comment:
check that is is a `mxc://` url?

Reply:
Is checking for that prefix sufficient?

Reply:
Yes, at least it will filter out simple error.
We already have code like that `.startsWith("mxc://")` elsewhere. We could probably create a dedicated `fun String.isMxc()` extension SDK side.